week-5/C-promises/exercise.js

```diff
@@ -45,6 +48,9 @@ function exercise3() {
 */
 function exercise4() {
   var promise4 = concatPromise()
+  promise4
+  .then(result => result + 'RAJ')
+  .then( conResult => document.querySelector('#exercise4').textContent = conResult);
```

Comment:
This is nice & clear what you're doing, I like it 👍 

There is another way of doing it (which isn't necessarily better, just letting you know), where you concatenate the string and update the DOM in 1 step:

```js
.then((result) => {
  document.querySelector('#exercise4').textContent = result + 'RAJ';
})
```

Reply:
I thought we were expected to familiar with promise chaining. 
 step 1: Take the result of promise4 and concatenate your name onto the end of it.
  step 2: Then return your concatenated string as a promise. (!!! )
  step 3: Finally insert the result  of your promise and insert it into the #exercise4 element

Reply:
This wasn't a criticism! Just pointing out that there's several ways of doing it. In general I would expect most people would use method I used because it's shorter, however I prefer your way :)